src/tests/sqlsmith/develop.md

```diff
@@ -5,6 +5,14 @@
 Suggest testing it with madsim, so you can reproduce the same results each time.
 See [Running with Madsim](./README.md#running-with-madsim).
 
+### Focused property testing
+
+When introducing a new feature, you can customize [config.yml](./config.yml) to
+set its generation weight and toggle its availability for targeted testing.
```

Comment:
We'd better be more precise and remark that it's a probability (0-100%) rather than an arbitrary weight.